pkg/fanal/image/daemon/podman_test.go

```diff
@@ -43,6 +43,7 @@ func setupPodmanSock(t *testing.T) *httptest.Server {
 }
 
 func TestPodmanImage(t *testing.T) {
+	t.Skip("podman.sock is not available for Windows CI")
```

Comment:
Looks like it skips anytime.

```suggestion
	if runtime.GOOS == "windows" {
		t.Skip("podman.sock is not available for Windows CI")
	}
```